### ISSUE TITLE
How to remove the pointer pressed animation in ListViewItem 

### ISSUE THREAD

**Issue by Subashini2129** [microsoft/microsoft-ui-xaml#5943]
Hi Team,

I need to remove the pointer pressed animation in ListViewItem. I have checked, but does not have any direct property to handle it in ListViewItemPresenter.

Can you please suggest any solution regarding this.

**Comment by StephenLPeters** [comment]
Hey! what animation are you referring to? The Tilt animations?

**Comment by Subashini2129** [comment]
@StephenLPeters, yes referring the Tilt animation only. Whether it is possible to disable it in ListView control.

**Comment by StephenLPeters** [comment]
The default style for ListView controls was changed with Windows 11 and will no longer have a tilt animation.  If you want to remove the tilt animation on windows 10 you need to apply a style which **does not** have the pointer theme animations that are present in the default style https://github.com/microsoft/microsoft-ui-xaml/blob/9fcf51815929ce6ca088cc22be7c14c7452edf0b/dev/CommonStyles/ListViewItem_themeresources_v1.xaml#L296  You can see we removed them in the V2 version of the style.

However ListViewItem doesn't utilize an exposed style by default, hence the ListViewItem**Expanded** title of the style.  Instead it generates the visuals from code behind because it is significantly more efficient and we expect there to be many, many list view items.  So be warned, if you do apply a custom style, like the V2 version of the ListViewItemExpanded style, that does remove the tilt for windows 10 apps you might lose some performance.  If you do decide to do this, it would be benefitial to only apply the new style on Windows 10 assuming the windows 11 style meets your requirements. 

@RBrid to confirm
